tests/src/test/java/alluxio/client/rest/AlluxioMasterRestApiTest.java

```diff
@@ -74,21 +74,21 @@ public void before() {
         .getMaster(FileSystemMaster.class);
     mHostname = sResource.get().getHostname();
     mPort = sResource.get().getLocalAlluxioMaster().getMasterProcess().getWebAddress().getPort();
-    mServicePrefix = AlluxioMasterRestServiceHandler.SERVICE_PREFIX;
+    mBaseUri = String.format("%s/%s", mBaseUri, AlluxioMasterRestServiceHandler.SERVICE_PREFIX);
```

Comment:
Don't want to be too picky on this, but generally speaking, child class should not modify a member variable of parent like this. I don't know why these all need to be done in a `@Before` method, but all member variable initialization should be done in constructors, and parent member variable initialization should be done in super(xxx) inside the constructor. This is not a specific comment to `mBaseUri` so it's a non-blocking comment for this PR.

Reply:
I'm not familiar with JUnit but it seems like the convention is to just use `@Before` over constructors anyway?
- https://stackoverflow.com/a/6094143

I spent a little bit of time trying to alter the implementations to use constructors rather than `@Before`, I think maybe it's because we throw exceptions in the `@Before` methods which are irrecoverable.

Reply:
For test classes this is probably ok. Generally speaking class interactions are more limited in tests. But in code, modifying a member variable not through an API in the same class defining the variable is just asking for trouble. So as long as we don't do that I'm fine.